### DISCUSSION TITLE
Copilot suggestions are glitchy

### DISCUSSION THREAD

**Discussion by kenan238** [community/community#19212] [Copilot Conversations]
![image](https://user-images.githubusercontent.com/73741117/175132696-3acc18be-3e51-472d-85f9-b6a77ee4cb47.png)
Here's an example
Sometimes my text just pushes it

**Comment by melvincwng** [comment]
@kenan238, you can report bugs to [GitHub Bug Report](https://support.github.com/contact/bug-report) over here.

Hope this clarifies.

**Reply by melvincwng** [reply]
@kenan238, if this reply has helped you, please kindly 'Mark as answer' so that others are aware this issue has been addressed.

Thanks, and have a good day :)

**Reply by kenan238** [reply]
Oh thanks!

**Reply by kenan238** [reply]
Never mind, bug report also has a bug that doesn't let me report.

**Reply by melvincwng** [reply]
@kenan238, ironically yes, the bug report form seems to have an issue...

**Reply by melvincwng** [reply]
@kenan238, alternatively try [Github Support](https://support.github.com/contact/account) instead. 

They usually respond back to you within a couple of days after investigating the issue.

Hope this helps.

If this is helpful, please 'Mark as answer' so that others know this issue has been addressed.

Thanks and have a nice day!

**Comment by melvincwng** [comment]
@kenan238, I have responded to your question that is in another thread as well.
